### PR TITLE
Fryske-Oranjekoeke V0.2

### PR DESCRIPTION
**Pull request by SanderT2001**
- Partials can now also be used in Controllers
- Added the UrlBuilder (Partial)
- Views can now use Monads for some set() functions
- The names of the Functions in the Content Partial are shorter, so that it is more readable in the views.
- Added Exception Handling with a small and very basic View

### CODE REVIEW

**Comment by SanderT2001** [comment]
- Partials can now also be used in Controllers
- Added the UrlBuilder (Partial)
- Views can now use Monads for some set() functions
- The names of the Functions in the Content Partial are shorter, so that it is more readable in the views.

File: app-skeleton/src/Views/Home/index.php [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<?= $this->Content->startContentBlock('main_content'); ?>
```
Comment (allartdo) [inline app-skeleton/src/Views/Home/index.php:1]: IMPROVEMENT!

File: src/Partial/UrlBuilder.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
+<?php
```
Comment (allartdo) [inline src/Partial/UrlBuilder.php:1]: Velli nice

**Review by allartdo — COMMENTED** [review]
Wai jou euhh remove me author man. I mutsj apprisieet mie kredits!

**Review by allartdo — COMMENTED** [review]
```
# Fryske Oranjekoeke
The simple MVC PHP Framework.

- ## Authors | Contributors
- * Sander Tuinstra - Author - [SanderT2001](https://github.com/SanderT2001)
- * Yvar Nanlohij   - Contributor - [Yvar-School](https://github.com/Yvar-School)
- * Allart de Jong  - Contributor - [allartdo](https://github.com/allartdo)
- * Justin Rijsdijk - Contributor - [justinrijsdijk](https://github.com/justinrijsdijk)

## Build with
* NGINX 1.16
* PHP 7.3.11
```

Wai jou euhh remove me author man. I mutsj apprisieet mie kredits!

**Comment by allartdo** [comment]
Ah woeps! Sorry i do it dooble instead of edit!

**Pull request merged**